script/plugin_webpack_directories.rb

```diff
@@ -31,8 +31,9 @@
   # skip the plugin if its path is already included
   next if config[:paths].include?(path)
   if File.exist?(entry)
-    config[:entries][bundle_name dep.name] = entry
+    config[:entries][bundle_name(dep.name)] = entry
```

Comment:
I think these shouldn't be calling `bundle_name` to fix what I mentioned in https://github.com/theforeman/foreman/pull/5413/files#r180416075 , otherwise I can't precompile other plugins but Katello.

The problem I found, however, is that `foreman_remote_execution_core` and `foreman_ansible_core` are picked up first